deployment/playbooks/metron_install.yml

```diff
@@ -75,7 +75,8 @@
     - { role: flume,  when: install_snort | default(True) == True }
     - { role: snort , when: install_snort | default(True) == True }
     - { role: yaf, when: install_yaf | default(True) == True }
-    - { role: pcap_replay , when: pcap_replay | default(False)  == True }
+    - { role: pcap_replay , when: (pcap_replay | default(False)) or (sensor_test_mode | default(False)) == True }
+    - { role: sensor-test-mode, when: sensor_test_mode | default(False) == True }
```

Comment:
Although sensor-test-mode has pcap_replay as a dependency in `deployments/roles/sensor-test-mode/meta/main.yml`, so maybe I do not need need the extra logic in `deployment/playbooks/metron_install.yml`?